lib/rsolr/uri.rb

```diff
@@ -47,12 +38,37 @@ def params_to_solr(params, escape = true)
   # query strings faster.  Use this rather than the cgi.rb
   # version since it's faster.
   # (Stolen from Rack).
+  # 
+  # 2015-02
+  # The Rack stuff is from 
+  #  http://www.rubydoc.info/github/rack/rack/URI.encode_www_form_component
+  # We instead will rely on 
+  #  http://ruby-doc.org/stdlib-2.2.0/libdoc/uri/rdoc/URI.html#method-c-encode_www_form_component
+  #  which is from the Ruby stdlib.  Hence this method is deprecated.
+  # @deprecated
   def escape_query_value(s)
```

Comment:
I'd just as soon remove it altogether.  The deprecation was to allow that to happen.  I already removed the calls to this method from within RSolr.